contrib/kind.yaml.j2

```diff
@@ -33,12 +34,14 @@ nodes:
    extraMounts:
      - hostPath: /tmp/kind
        containerPath: /var/run/secrets/kubernetes.io/serviceaccount/
-# - role: worker
-#   extraMounts:
-#     - hostPath: /tmp/kind
-#       containerPath: /var/run/secrets/kubernetes.io/serviceaccount/
-# - role: worker
-#   extraMounts:
-#     - hostPath: /tmp/kind
-#       containerPath: /var/run/secrets/kubernetes.io/serviceaccount/
+{%- else %}
+ - role: worker
```

Comment:
would be kind of cool to add a new cli argument to kind.sh -n "number of worker nodes, default 2 for noha":
https://github.com/ovn-org/ovn-kubernetes/blob/master/contrib/kind.sh#L27

and then here you can just do a for number of worker nodes ...

Reply:
I'll add a followup PR with this enhancement.